### PR TITLE
tools: test: topology: add argument to fix build error in test tplg

### LINKED ISSUE #1204: Tool test topology build failed
m4: test-tone-playback.m4: 45: fatal error: Invalid parameters (8) to PIPELINE_ADD
m4: test-tone-playback.m4: 45: fatal error: Invalid parameters (8) to PIPELINE_ADD
m4: test-tone-playback.m4: 45: fatal error: Invalid parameters (8) to PIPELINE_ADD
test/topology/CMakeFiles/tests.dir/build.make:106: recipe for target 'test/topology/CMakeFiles/tests' failed
make[3]: *** [test/topology/CMakeFiles/tests] Error 123
CMakeFiles/Makefile2:1557: recipe for target 'test/topology/CMakeFiles/tests.dir/all' failed
make[2]: *** [test/topology/CMakeFiles/tests.dir/all] Error 2
CMakeFiles/Makefile2:1564: recipe for target 'test/topology/CMakeFiles/tests.dir/rule' failed
make[1]: *** [test/topology/CMakeFiles/tests.dir/rule] Error 2
Makefile:697: recipe for target 'tests' failed
make: *** [tests] Error 2

### PR DESCRIPTION
**Pull request by xun2z**
remove test-tone-playback in test to fix build errors in ci

This is to fix #1204 

Signed-off-by: Xun Zhang <xun2.zhang@intel.com>

### CODE REVIEW

**Comment by xiulipan** [comment]
@ranj063 @lgirdwood 
Fix #1204

File: tools/test/topology/tplg-build.sh [hunk 1]
```diff
@@ -20,7 +20,6 @@ fi
 # Simple component test cases
 # can be used on components with 1 sink and 1 source.
 SIMPLE_TESTS=(test-all test-capture test-playback)
-TONE_TEST=test-tone-playback
```
Comment (jajanusz) [inline tools/test/topology/tplg-build.sh]: I think that if you remove that variable, you should also change lines that use removed variable, f.e: `simple_test codec tone "SSP2-Codec" s32le SSP 2 s16le 20 16 1920000 19200000 I2S 0 TONE_TEST[@]`

**Comment by ranj063** [comment]
@xiulipan @xun2z sorry this isnt the right fix. I think the error in #1204 is pretty clear. The number of arguments in test-tone-playback.m4 for PIPELINE_ADD is insufficient. Can you please add a 0 to the last argument to fix this.

**Comment by xun2z** [comment]
> Can you please add a 0 to the last argument to fix this.

@ranj063 you are right. Add a 0 can simply fix this error. @jajanusz I've changed as Ranjani commented

**Comment by xiulipan** [comment]
@ranj063 
What I worry here is that we have removed support of tone in #925 
Do we still need to support the tone related part in test topology?

**Comment by ranj063** [comment]
> @ranj063
> What I worry here is that we have removed support of tone in #925
> Do we still need to support the tone related part in test topology?

@xiulipan we removed tone pipelines from top level m4 because it isnt tested. And if someone were to test it, they would want to start with the test topology.

**Comment by lgirdwood** [comment]
@xiulipan this will let CI continue until you remove the test as @singalsu does not use them.

**Comment by xiulipan** [comment]
@ranj063 
I think let keep the test topology here and make it build. But I am very sure most of the codec topology won't work now.
@lgirdwood 
Should we still check the build status of the test topology now?

**Comment by lgirdwood** [comment]
@xiulipan yes, we must check build status of everything

**Pull request merged**